packages/eslint-plugin-okta/package.json

```diff
@@ -8,6 +8,11 @@
     "test": "jest --config jest.conf.js",
     "test-report": "yarn test --testResultsProcessor=jest-junit"
   },
+  "repository": {
```

Comment:
I don't this is needed in this repo as it cross references a different repo from here. And we are retiring this. @shuowu-okta